package.json

```diff
@@ -105,7 +105,7 @@
     "react-native-listener": "1.0.1",
     "react-phone-number-input": "^0.15.2",
     "react-redux": "4.4.6",
-    "react-webcam-onfido": "0.0.20",
+    "react-webcam-onfido": "0.1.0-rc.1",
```

Comment:
Why are we using rc? Don't we have a 0.1.0 version?

Reply:
I will make a release once I get approved, I won't change the code.